app/lib/event_bus.rb

```diff
@@ -36,6 +36,11 @@ def self.reset!
   ReindexAndNotifyFilterSubscriptionsJob.perform_later(thread.id)
 }
 
+# reindexing
+EventBus.subscribe :message_changed, ->(message) do
+  Searchable::ReindexMessageThreadJob.perform_later(message.thread.id)
+end
+
 # reindexing on removals
 EventBus.subscribe :tag_renamed, ->(tag) do
   ReindexAndNotifyFilterSubscriptionsJob.perform_later_for_tag_id(tag.id)
```

Comment:
Napisal som to sprosto, presne naopak som to myslel. Oneliner v klude daj ako { } ako je vsetko nizsie.